tools/cli/commands.js

```diff
@@ -660,8 +660,8 @@ main.registerCommand({
 
       // We do mind if there are non-hidden directories, because we don't want
       // to recursively check everything to do some crazy heuristic to see if
-      // we should try to creat an app.
-      var stats = files.stat(filePath);
+      // we should try to create an app.
+      var stats = files.stat(files.pathJoin(appPath, filePath));
```

Comment:
I know this code used `files.stat` unsafely already, but perhaps we should use `files.statOrNull` and then `stats && stats.isDirectory()` below, just in case the new path does not exist?